packages/trpc/server/routers/viewer/teams.tsx

```diff
@@ -149,17 +149,28 @@ export const viewerTeamsRouter = createProtectedRouter()
         },
       });
 
+      if (!prevTeam) throw new TRPCError({ code: "NOT_FOUND", message: "Team not found." });
+
+      const data: Prisma.TeamUpdateArgs["data"] = {
+        name: input.name,
+        logo: input.logo,
+        bio: input.bio,
+        hideBranding: input.hideBranding,
+      };
+
+      if (
+        input.slug &&
+        IS_TEAM_BILLING_ENABLED &&
+        /** If the team doesn't have a slug we can assume that it hasn't been published yet. */ !prevTeam.slug
+      ) {
+        data.metadata = { requestedSlug: input.slug };
```

Comment:
Shouldn't we throw an error here, that you can't add a slug without publishing? This is silently discarding the slug which user can consider a bug

Reply:
Could be. Also we would need a way to publish from the teams view

Reply:
Added error instead